htdocs/modules/system/class/block.php

```diff
@@ -625,7 +630,7 @@ public function countSimilarBlocks($moduleId, $funcNum, $showFunc = null)
             // invalid query
             return 0;
         }
-        /* @var XoopsMySQLDatabase $db */
+        /* @var \XoopsMySQLDatabaseProxy $db */
```

Comment:
Scrutinizer was complaining about it, so I've tried to change it to see if it helps. 
It was late at night and I didn't finish it, so feel free to close it, unless you see how it could be fixed.

Reply:
Let's see how #754 looks, and then decide how to go.

Reply:
I think #754 achieves the same goal. If you find it acceptable, we can merge it and close this one.